workloads/kube-burner-ocp-wrapper/run.sh

```diff
@@ -74,7 +84,7 @@ if [[ ${WORKLOAD} =~ "cluster-density" ]]; then
   ITERATIONS=${ITERATIONS:?}
   cmd+=" --iterations=${ITERATIONS} --churn=${CHURN}"
 fi
-if [[ -n ${MC_KUBECONFIG} ]]; then
+if [[ -n ${MC_KUBECONFIG} ]] && [[ -n ${ES_SERVER} ]]; then
```

Comment:
By the way, how does the `MC_KUBECONFIG` kubeconfig get passed here?

Reply:
MC_KUBECONFIG is a variable set by the user